spec/models/post_spec.rb

```diff
@@ -0,0 +1,21 @@
+RSpec.describe Post, type: :model do
+    subject { FactoryBot.create(:post) }
```

Comment:
Unnecessary subject while using shoulda matchers. Add another test to check this model has valid factory

Reply:
removed subject.